lib/data/parse-test-csv-row.js

```diff
@@ -203,25 +204,28 @@ function parseTestCSVRowV2({ tests, assertions, scripts, commands }) {
       return (
         testId === commandInfo.testId &&
         command === commandInfo.command &&
-        settings === commandInfo.settings &&
+        settings === commandInfo.settings && // the primary settings value
         presentationNumber === commandInfo.presentationNumber
       );
     }
 
     for (const command of commands) {
       testsParsed.forEach(test => {
+        const [primarySetting, ...additionalSettings] = command.settings
+          ? command.settings.split(' ')
+          : ['defaultMode'];
+
         if (
           command.testId === test.testId &&
-          test.target.ats.some(
-            at => at.key === key && at.settings === (command.settings || 'defaultMode')
-          )
+          test.target.ats.some(at => at.key === key && at.settings === primarySetting)
         ) {
           const commandInfo = {
             testId: command.testId,
             command: command.command,
-            settings: command.settings || 'defaultMode',
+            settings: primarySetting,
```

Comment:
This definition of `settings` isn't really appropriate in the presence of `additionalSettings`. Will this change also be reflected in [the test format definition](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2)? What would be the ramifications of renaming this property to `primarySetting`?

Reply:
> This definition of `settings` isn't really appropriate in the presence of `additionalSettings` ... renaming this property to `primarySetting`?

I agree! I figured as much when sharing the initial [gist on what's different here](https://gist.github.com/howard-e/89a06d917946f5e922740dccee08d4e2). I proposed `mainSetting` then and have since used `primarySetting` here as well which feels more appropriate. Otherwise, it certainly presents a bit of confusion without this context.

> Will this change also be reflected in [the test format definition](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2)?

I don't think it necessary. Seems to be something to consider as an implementation detail.

> What would be the ramifications ...

A widespread refactor across the references in this repository, aria-at-app (as well as some migration or backward compatibility work) and also in aria-at-automation-harness and related resources I assume. That will certainly need a bit of coordination that I'm uneasy trying to get that done now. Do you feel strongly otherwise?

Reply:
I agree that it's an implementation detail for the Community Group at large. That said, the amount of inter-project coordination needed to support change makes it feel like a public-facing design.

This change creates a gap between what the CG calls "settings" and what code maintainers call "settings", and to me, that's a problem which is distinct from the internally inconsistent names.

I dropped in from https://github.com/w3c/aria-at-automation-harness/pull/73 to verify that my understanding of the situation was accurate. I trust your judgement when it comes to coordinating changes; I'd only advocate for prioritizing that refactoring sooner rather than later!